events/2018/05-contributor-summit/README.md

```diff
@@ -20,29 +20,31 @@ In some sense, the summit is a real-life extension of the community meetings and
 - Tuesday, May 1, 2018 (before Kubecon EU)
 - Bella Center
 - Copenhagen, Denmark
-- There is also a [Slack channel](https://kubernetes.slack.com/messages/contributor-summit) for you to use during the summit itself to pass URLs, etc. 
+- There is also a [Slack channel](https://kubernetes.slack.com/messages/contributor-summit) for you to use during the summit itself to pass URLs, etc.
 
 All day event with a happy hour reception to close.
 
-## Agenda 
+## Agenda
 
 ### Morning
 
 | Time        | Track One                       | Track Two                     | Track Three    |
-| ----------- | ------------------------------- |  ---------------------------- | -------------- | 
+| ----------- | ------------------------------- |  ---------------------------- | -------------- |
 | 8:00        | Registration and Breakfast      |                               |                |
 | 9:00-9:15   | Welcome and Introduction        |                               |                |                   
 | 9:15-9:30   | Steering Committee Update       |                               |                |
 |             |                                 |                               |                |
-|             | New Contributor Workshop        | Current Contributor Workshop  | Docs Sprint    |
+|             | [New Contributor Workshop](https://github.com/kubernetes/community/tree/master/events/2018/05-contributor-summit/new-contributor-workshop.md)        | Current Contributor Workshop  | Docs Sprint    |
```

Comment:
just use `/events/2018/05-contributor-summit/new-contributor-workshop.md` for the link

Reply:
ah, wasn't sure that would work.